extensions/sql-database-projects/package.json

```diff
@@ -477,7 +477,7 @@
         },
         {
           "command": "sqlDatabaseProjects.createProjectFromDatabase",
-          "when": "!azdataAvailable && view == objectExplorer && viewItem =~ /^(disconnectedServer|Server|Database)$/",
```

Comment:
I believe viewItem is not supporting in vscode, if this is an intentional change or not, removing this condition still works as expected for both ADS and vscode, the item is visible for disconnectedServer/Server and Database nodes.